src/main/java/com/example/cat/CatController.java

```diff
@@ -0,0 +1,18 @@
+package com.example.cat;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/API")
+public class CatController {
+
+            //This is a GET request that returns a Cat object
+            @GetMapping("/miauz")
+            public Cat GETCAT() {
+                Cat MIAUZ = new Cat("Garfield", 39);
```

Comment:
Wir solltest die Cat nicht hard-coden sondern dynamisch aus dem Service abrufen
```suggestion
                Cat requestedCat = new Cat("Garfield", 39);
```